numba/np/arrayobj.py

```diff
@@ -4958,12 +4958,100 @@ def np_flip(a):
         raise errors.TypingError("Cannot np.flip on %s type" % a)
 
     def impl(a):
-        sl = _build_slice_tuple(a.ndim)
+        sl = _build_flip_slice_tuple(a.ndim)
         return a[sl]
 
     return impl
 
 
+@overload(np.array_split)
```

Comment:
`np.array_split` and `np.split` should be added to the supported functions list in the docs here: https://github.com/numba/numba/blame/master/docs/source/reference/numpysupported.rst#L330 thanks

Reply:
thanks